docs/specs/editorconfig-in-the-compiler.md

```diff
@@ -0,0 +1,111 @@
+All-capital words are interpreted per [RFC 2119](https://www.ietf.org/rfc/rfc2119.txt).
+
+# Compiler Command Arguments
+
+The compiler *MUST* receive a new switch, `/editorconfig`, which accepts a single
+file which is an .editorconfig file that may apply to one or more files in the
+compilation. Multiple files are passed with multiple `/editorconfig` switches,
+the same way you can pass multiple references with /reference or multiple analyzers
+with /analyzer. There is no short form of the argument. Files passed with relative
```

Comment:
@tannergooding 

I understand the established patterns and yes if everyone followed them life would be grand.  The compiler though doesn't get to deal with everyone doing the right thing.  We get to deal with all of the cases when people didn't do the right thing.  

The ability of a stray file, completely outside your repo and compilation, to effect the outcome of a compilation by default is a non-starter for us.